### ISSUE TITLE
updates to Sierra this morning on MBA mid 2011, prior to updating CAT was working flawlessly, but now i cant get it to work with my iphone6 with ios10, im not sure if its an ios10 issue now as I only updated to that this morning also. 

### ISSUE THREAD

**Issue by digger99** [dokterdok/Continuity-Activation-Tool#335]
Have you tried [these steps](https://github.com/dokterdok/Continuity-Activation-Tool/wiki#cant-open-the-tool)?

Is there already an open topic for this issue?

Please include the following information.

- OS Version:
- Mac model:
- Dongle or replaced card?:
- Used dongle/card:

Your Issue:

Steps done to try to fix the issue:

**Comment by digger99** [comment]
Ive tried numerous times over the weekend, tried uninstalling it, and reinstalling but to no avail, Im on very same software versions as you are, also have macbook air mid 2011 and iphone 6 running 10.0.1. I was able to confirm that its not an ios10 issue as I uninstalled sierra and reverted back to El capitan and handoff worked directly from my back up despite my phone now been on ios10.

**Comment by durosity** [comment]
"Has anyone here gotten it to work fully on their mid-2011 MBA with Sierra GM & iOS 10 GM?" 

I have on a 2011 mac mini which has the same card as the 2011 MBA, so it definitely should work.  Are they both on the same iCloud account?  Also i remember someone once reporting a similar issue where creating a brand new Mac user account fixed the problem for them so might be worth signing out of icloud on the Mac, create a new admin user, then log back in again.

**Comment by miguelmanalo** [comment]
@digger99 Ah, so you think it's a new issue with Sierra that the tool needs to be updated for?

@durosity Thank you! Yes, both devices are on the the same iCloud account. You're having success with pairing the Sierra GM & iOS 10 GM with your Mac Mini set up? With Handoff, phone calls, clipboard and everything?

**Comment by digger99** [comment]
@miguelmanalo Well, thats what it looks like to me anyway, but Im open to trying out other things if others have alternative solutions for me to try

**Comment by miguelmanalo** [comment]
@digger99 Your conclusion makes sense to me, thank you! I'll keep my fingers crossed that after Sierra & iOS 10 are officially released this tool still can work for us with updates.

**Comment by durosity** [comment]
@miguelmanalo Yeah everything seems to work fine except watch unlocking, i suspect due to the 'time of flight' requirements.  So it's a bit strange that it's not working on your MBA.. guess there must be a minute difference between them that's causing the issue.

**Comment by miguelmanalo** [comment]
@durosity @digger99 OMG I did something weird and now Handoff is working!! https://imgur.com/a/CdzjU

Clipboard sharing, Safari/iMessage handoff!

What I did is:

- Settings > iCloud > iCloud Drive > Toggle Off > Toggle On > Stuff works

**Comment by alecnash** [comment]
Have the same issue. MBA mid 2011. MacSierra 10.12 and iOS 10.0.1 iPhone 6s. The Toggle didn't do the trick for me.

**Comment by frizby66** [comment]
Toggle worked for me..MBP early 2011 with BC card.

**Comment by digger99** [comment]
Same setup here for me as @alecnash has and the toggle didn't work for me either.

**Comment by seargentpepper** [comment]
I did "off/on" but failed again. I can't do that everyday.

**Comment by libertyforall1776** [comment]
I am having the same issue, but on 10.12.1 Beta 5 (16B2553a)

**Comment by digger99** [comment]
Updated my iphone 7 to ios10.2 beta on thursday night and Ive just noticed that handoff is now working again for me, happy days.
